config/routes.rb

```diff
@@ -11,7 +11,13 @@
     resources :dashboard, only: [:index]
     resources :loans
     resources :organizations
-    resources :project_steps
+    resources :project_steps do
+      collection do
+        delete '', to: 'project_steps#batch_destroy'
+        patch 'adjust_dates', to: 'project_steps#adjust_dates'
+        patch 'finalize', to: 'project_steps#finalize'
+      end
```

Comment:
`delete :batch_destroy` didn't work.  Is there a cleaner way to map the empty subpath to an controller method?